content/post/web-analytics-process.md

```diff
@@ -0,0 +1,83 @@
+---
+author: Ana Kravitz
+title: "Web Analytics Maintenance Process"
+publishDate: "2018-07-28"
+draft: true
+tags: 
+ - web analytics
+ - tracking plan
+ - event tracking
+---
+
+A web analytics implementation project often starts with quite a lot of fanfare and resources. There will usually be an audit and needs assessment process to determine what tracking needs to be implemented or fixed, an implementation project plan identifying task owners and dates, and earmarked hours from the development team for tasks like implementing tracking code and building a data layer. All of this generally ensures that there is satisfactorily comprehensive and accurate tracking in place at the end of the project. So why do we still regularly see web analytics issues?
+<!--more-->
+
+Common web analytics issues:
+* Broken tracking
+* Unexplained data discrepancies
+* Untagged marketing campaigns or site elements
+* Confusing and messy naming conventions
+* Etc…
+
+## Why web analytics breaks
+Web analytics tracking typically relies on front-end technology like HTML, CSS, and JS, and therefore is prone to break as the site structure changes. This means that for most organizations, their web analytics implementation deteriorates over time as the site gets updated and attention shifts to other projects. 
+
+This is especially true in the common case where web analytics does not clearly belong to any one department. Frequently, Marketing and Product teams are the primary consumers of the data, but lack the technical skills to implement tags themselves. In this case they either need to turn to the IT team, or rely on outside agencies and consultants for planning and implementation. 
+
+This mismatch between who uses the data and who implements data collection often results in a poor feedback cycle and lengthy delays. Without constant vigilance, site changes degrade tags, new features do not always get tagged, and so on. This often goes unnoticed until someone tries to dig into the data and discovers the problem, at which point there is a flurry of activity to resolve the issue (depending on how serious it is, the entire implementation process may start again). 
+
+In this post I will provide a series of steps to follow to make sure a web analytics tool is not just implemented correctly but also maintained over time.
+
+## Web Analytics Maintenance Process
+### Step 1: Audit the existing implementation
+If you already have a web analytics tool in place, methodically review the account setup and run a technical tracking review. 
+
+The audit should include the following elements, at least:
+* All site tracking tags (standard pageview tags as well as any custom tags) to ensure they’re firing correctly
```

Comment:
The second clause here "to ensure they’re firing correctly" reads a bit weird because it's not clear what the verb here is. I think it's "audit" or "review" but that doesn't work given the way the sentence introducing the  list is phrased.

Reply:
I changed it to "The following elements should be audited", please let me know if that works better